docs/dependency_confusion.md

```diff
@@ -0,0 +1,67 @@
+# Problem Statement
+
+Recent research has shown novel [ways](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610) to inject malicious content into applications. The question has been raised as to whether or not using Cachito makes this issue worse. This document is an analysis of each package manager supported by Cachito to address the security concern named *dependency confusion*.
+
+# Dependency Confusion
+
+To paraphrase the meaning of *[dependency confusion](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610)*: a package manager is tricked into installing a dependency from an official, public, repository when it should’ve installed it from a custom, often private, repository. For example, a company may host their own internal npm registry which hosts internal-only packages. Then, someone on the Internet uploads a package of the same name to the official npm repository. When the application is installed, the package manager chooses the dependency package from the public repository instead of the internal one.
```

Comment:
```suggestion
To paraphrase the meaning of *[dependency confusion](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610)*: a package manager is tricked into installing a dependency from an official, public, repository when it should’ve installed it from a custom, often private, repository. For example, a company may host its internal npm registry which hosts internal-only packages. Then, someone on the Internet uploads a package of the same name to the official npm repository. When the application is installed, the package manager chooses the dependency package from the public repository instead of the internal one.
```

Reply:
Or maybe I am wrong :D

Reply:
You're right, I'll make that change!